test/integration/master/test_build_artifact.py

```diff
@@ -33,3 +43,19 @@ def test_update_timing_file(self, existing_timing_data, new_timing_data, expecte
             updated_timing_data = json.load(timing_file)
 
         self.assertDictEqual(updated_timing_data, expected_final_timing_data)
+
+    def test_get_failed_subjob_and_atom_ids_returns_correct_ids(self):
+        """
+        Build artifact directory:
+            artifact_1_0/clusterrunner_exit_code -> 0
+            artifact_1_1/clusterrunner_exit_code -> 1
+            artifact_2_0/clusterrunner_exit_code -> 0
+            artifact_2_1/clusterrunner_exit_code -> 1
+
+        Expected to return: [(1,1), (2,1)]
+        """
```

Comment:
There's this kind of weird behavior of nosetests where when you add a docstring to a test, it uses the first line as the test name in verbose output.
You can see it on line 736 in the unit tests: https://travis-ci.org/box/ClusterRunner/jobs/77419336#L736

Probably the easiest workaround is just to make this a normal comment. Actually the easiest thing is just to ignore it... :sunglasses: